Modules/Core/Mesh/wrapping/test/itkMeshArrayPixelTypeTest.py

```diff
@@ -27,7 +27,13 @@
 mesh = MeshType.New()
 
 # Create Vector Container and Store values in it for each Point
-v = itk.VectorContainer[itk.UL, PixelType].New()
+# For windows use itk.ULL
+if hasattr(itk.VectorContainer, 'ULAD'):
```

Comment:
A generic way to check for Windows is `if os.name == 'nt'`. But if the one you wrote here works, we should keep it.